release-notes/swan-lake-alpha3-release-note.md

```diff
@@ -140,14 +140,32 @@ xml:Text nameText = (name/*).text();
 io:println(nameText); // "DanBrown"
 ```
 
+##### Bug Fixes
+
+To view bug fixes, see the [Github milestone issues](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+is%3Aclosed+milestone%3A%22Ballerina+Swan+Lake+-+Alpha3%22+label%3AType%2FBug+label%3ATeam%2FCompilerFE).
```

Comment:
These seem to be different from the template?

Reply:
Sorry, what is the difference?

Reply:
Fixed the SLAlpha3 release note also accordingly.